saleor/invoice/emails.py

```diff
@@ -23,7 +25,11 @@ def collect_invoice_data_for_email(invoice_pk, template):
 
 
 @app.task
-def send_invoice(invoice_pk):
+def send_invoice(invoice_pk, user_pk):
```

Comment:
Maybe we should use more specific name for `user_pk` like `staff_user_pk` or `requester_user_pk`. 
Another solution is add docstring with args descriptions. 

Currently, I don't know it a customer pk or staff user pk.